### ISSUE TITLE
Am not able to view progress using screen reader.

### ISSUE THREAD

**Issue by Computerfan23** [DavidSM64/Diddy-Kong-Racing#369]
Hi there!
I am a blind gamer and i use a screen reader and tts to view project sites. I am unable to view the progress for dkr decomp since screen readers are unable to read percent if it is in certain formats.
Would it be possible to fix this? By perhaps have the progress percentage be shown as text?
Thanks in advance!

**Comment by Ryan-Myers** [comment]
Well that's an interesting problem. I assumed because we had it in plain text it would be legible for you. The current progress is 49.78 percent complete and it's formatted like this:
`------------------------- 49.78% Complete -------------------------`

What could we do to help you out?

**Comment by Computerfan23** [comment]
Would it be possible to add it in plain text under the read me part? I had a similar issue for the banjo kazooie decomp.
It is easier for me to read the percentage in plain text since screen readers cannot handle images.

**Comment by Ryan-Myers** [comment]
Hmmm. We don't us images, is it possible the screen reader can't handle code blocks? We could look at putting the progress of just the main completion percentage outside of code blocks if it will help.

**Comment by DavidSM64** [comment]
It shouldn't be too difficult to do what Banjo Kazooie does. I'm surprised that the screen reader doesn't work with code blocks, since it is just normal text with hyphens around it. We'll try to solve it as soon as possible.

**Comment by DavidSM64** [comment]
> Would it be possible to add it in plain text under the read me part? I had a similar issue for the banjo kazooie decomp. It is easier for me to read the percentage in plain text since screen readers cannot handle images.

How does your screen reader work for percentages exactly? We technically have 2 different percentages, one for the actual decomp and another for documentation. I'm wondering if adding both of those would mess it up in any way.

**Comment by Computerfan23** [comment]
Yeah, screen readers are unable to handle code blocks. I get nothing read at all.
Regarding the percentage, i think the overall percentage should be enough.

**Comment by Computerfan23** [comment]
I think it would work for me to read the two different percentages if they are clearly labeled. Like decomp percentage, then next line could be document percentage or similar. Screen reader is able to read it otherwise fine when i check project like banjo kazooie decomp.

**Comment by tonyspumoni** [comment]
hey @Computerfan23 - you should be able to preview the new README from that PR by using the "Display rich diff" function from the "files changed" tab. Could you let me know if that works with your screen reader?

**Comment by Computerfan23** [comment]
I just tried doing that in the pr. Pressed the display diff button and i was able to preview it. My screen reader was able to read both percentage numbers. So it definitely works. Thanks for fixing this. Will make it much easier for me.
